src/components/10-atoms/spinner/README.md

```diff
@@ -0,0 +1,30 @@
+# Spinner
+
+Adds a loading spinner that can be used while something is loading.
+
+```bash
+npm install @axa-ch/spinner
+```
+
+```js
+import '@axa-ch/spinner';
+...
+<axa-spinner></axa-spinner>
+```
+
+## Properties
+
+### Variant
+
+| Attribute     | Details                   |
+| ------------- | ------------------------- |
+| `small=false` | Sets the spinner to small |
```

Comment:
Just call it `small` (without the false) and write the default value in the details section.

Reply:
Also mention that this is a boolean attribute.